File: Retos/Reto #0 - EL FAMOSO FIZZ BUZZ [Fácil]/python/RiveroDev.py

def  fizzBazz ():
    for  number  in  range ( 1 , 101 ):
        if  number  %  3  ==  0  and  number  %  5  ==  0 :
            print("fizzbazz" )
        elif number %  5  ==  0 :
            print( "bazz" )
        elif number  %  3  ==  0 :
            print( "fizz")
        else :
            print(number)
    
    
if  __name__ == "__main__" :

    fizzBazz ()

File: Retos/Reto #0 - EL FAMOSO FIZZ BUZZ [Fácil]/rust/RiveroDev.rs

// creamos la funcion principal
fn main(){
    for number in 1..101 {  // creamos un ciclo for que va de 1 al 100
        fizzbuzz(number)   // llamamos la funcion para que se ejecute con cada numero
    }
}

fn fizzbuzz(number: i32){

    match (number % 3 == 0, number % 5 == 0 ){ // usamos match y creamos la estructura comparativa
        (true,true) => println!("fizzbuzz"),
        (false,true) => println!("buzz"),
        (true,false) => println!("fizz"),
        _ => println!("{}",number),
    }
}